python/cuml/tests/dask/test_dask_logistic_regression.py

```diff
@@ -20,6 +20,8 @@
 from sklearn.datasets import make_classification
 from sklearn.linear_model import LogisticRegression as skLR
 from cuml.internals.safe_imports import cpu_only_import
+from hypothesis import given
+from hypothesis import strategies as st
```

Comment:
Please remove those imports unless you actually use them.

Reply:
Good catch. Removed!